week-1-practise/level-1/8-function-output-1.js

```diff
@@ -1,12 +1,16 @@
 function getNumber() {
-    return Math.random() * 10;
+  return Math.random() * 10;
 }
 
 var result = getNumber();
 console.log(result);
 
 // Answer these questions:
 // - What does this program do?
+// this program generates a random number whenever it is called between 0 to 1.
```

Comment:
It it between 0 and 1?

Hint: `Math.random()` produces a number between 0 and 1 but `getNumber` does not